### ISSUE TITLE
Windows file system paths fail to create/find step1

### ISSUE THREAD

**Issue by AlexBarbera** [labsyspharm/beta-segmentinator#1]
Input:

`Namespace(input='F:\\orionCrop2-1.tif', output='F:\\', thres_nms=0.1, thres_prediction=0.6, thres_mask=0.95, thres_size=0.3, tile_size=128, model_path='V:\\UnMICSTdev\\FOR ALEX HUMAN ANNOTATIONS USE THESE\\models_that_work\\model_trained_without_ignore_manyWindowsAndSynthetic.pt', device='cuda:0', rolling_window=10, parallel=False, no_viewer=False)
`

Error:
```
Traceback (most recent call last):
  File "C:\Users\Public\Documents\beta-segmentinator\main.py", line 410, in <module>
    pipeline(args)
  File "C:\Users\Public\Documents\beta-segmentinator\main.py", line 298, in pipeline
    if len(os.listdir(os.path.join(args.output, "step1"))) == 0 or True:
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'F:\\step1'
```

Running in windows

**Comment by AlexBarbera** [comment]
Actual input for output is `F:\`

**Comment by AlexBarbera** [comment]
Issue was no `mkdir` for step1, now solved in commit a099704e347718e2cd7815a6115b6824bb49fe1b
